File: tabuada.py
while True:
    numero = int(input("Digite um número para ver a tabuada: "))
 
    print(f"\n=== Tabuada do {numero} ===")
    for i in range(1, 11):
        print(f"{numero} x {i:2} = {numero * i}")
 
    opcao = int(input("\nDeseja ver outra tabuada? (1 = Sim / 0 = Não): "))
    if opcao == 0:
        print("Encerrando o programa. Até logo!")
        break
    print()

File: tabuada.c
#include <stdio.h>
 
int main() {
    int numero, i, opcao;
 
    do {
        printf("Digite um numero para ver a tabuada: ");
        scanf("%d", &numero);
 
        printf("\n=== Tabuada do %d ===\n", numero);
        for (i = 1; i <= 10; i++)
            printf("%d x %2d = %d\n", numero, i, numero * i);
 
        printf("\nDeseja ver outra tabuada? (1 = Sim / 0 = Nao): ");
        scanf("%d", &opcao);
        printf("\n");
 
    } while (opcao == 1);
 
    printf("Encerrando o programa. Ate logo!\n");
    return 0;
}